bluebrain/repo-bluebrain/packages/py-luigi-tools/package.py

```diff
@@ -10,9 +10,9 @@ class PyLuigiTools(PythonPackage):
     '''Tools to work with luigi.'''
 
     homepage = 'https://github.com/BlueBrain/luigi-tools'
-    url      = 'https://pypi.io/packages/source/l/luigi-tools/luigi-tools-0.0.15.tar.gz'
+    url      = 'https://pypi.io/packages/source/l/luigi-tools/luigi-tools-0.0.16.tar.gz'
```

Comment:
```suggestion
    pypi     = 'luigi-tools/luigi-tools-0.0.16.tar.gz'
```
Should work, too. And the version number does not need to be kept in sync with the defined ones :)

Reply:
yeah, just as we don't accumulate multiple versions, I find it nicer to keep them in sync :)

Reply:
Fair enough, less confusion probably. Now that I've disentangled repositories, I don't expect merge conflicts from it anymore, either.